R/mechanism.R

```diff
@@ -43,7 +43,7 @@ mechanism <- setRefClass(
         var.type.orig = 'character',
         n = 'numeric',
         epsilon = 'numeric',
-        delta = 'numeric',
+        delta = 'ANY',
```

Comment:
Can we add a comment clarifying that this is because delta is sometimes set to NULL? Every time I look at it I forget why it's not just numeric.